### ISSUE TITLE
Settable MAXIMUM_MODEL_AGE

### ISSUE THREAD

**Issue by Flix6x** [OpenSTEF/openstef#442]
The `MAXIMUM_MODEL_AGE` is hardcoded to 7 days (in `pipeline/train_model.py`). While I consider this a reasonable default, I would find it useful to be able to override the default by setting a different age explicitly, to support assets that require more frequent or less frequent retraining.

Could this be done, for example, by introducing it as a variable in the function signature of `train_model_pipeline()`? Or perhaps you'd consider it more suitable as a `PredictionJobDataClass` attribute?

**Comment by FrankKr** [comment]
Good idea, thanks for suggesting. I have added the label 'good first issue' to it.

**Comment by TheoPascoli** [comment]
Hello, is this issue still pending, resolved, or is it no longer needed?

**Comment by Flix6x** [comment]
From my end, still pending. I could work on it, provided I get some input on the desired implementation approach (see my previous question).

**Comment by TheoPascoli** [comment]
I was asking because I wanted to do it myself. I don't necessarily need this development to be done, but since it is labeled as a 'good first issue,' I thought it would be a good entry point to the repository for me, especially if there are other developments to be done later on from the RTE side. I'll try something then

**Comment by JonitaRuiter** [comment]
Hi Theo, sorry for the late reply! I think this issue is still pending, so if you can propose a PR, I'll make sure it is reviewed!

**Comment by egordm** [comment]
This is actually possible in new openstef 4. 

See this api reference page: https://openstef.github.io/openstef/api/generated/openstef_models.integrations.mlflow.MLFlowStorageCallback.html
